spectractor/simulation/adr/adr_utils_Spectractor.py

```diff
@@ -0,0 +1,97 @@
+import matplotlib.pyplot as plt
+import astropy.coordinates as AC
+from astropy import units as u
+import spectractor.simulation.adr.adr as ADR
+import numpy as np
+from spectractor import parameters
+
+"""
+Functions to:
+  - get adr object from fits file
+  - get list of shift due to adr corresponding to a list of lambdas
+  - get maximum shift due to adr between (in th range of lambdas given)
+  - plot the adr along both axis and the biggest one against the wavelength
+Example to get shift in pixels:
+  instanciation_adr(fitsfile, latitude, lbda_ref)
+  xs, ys = get_adr_shift_for_lbdas(adr_object, lbdas)
+  xs_pix, ys_pix = in_pixel(xs, fitsfile), adru.in_pixel(ys, fitsfile)
+"""
+
+__author__ = "Maxime Rey"
+
+#adapted by Vincent Brémaud
+
+
+# ================================= #
+#          Main functions           #
+# ================================= #
+
+
+def adr_calib(lambdas,params,lambda_ref=550):
+
+    lat=AC.Latitude(parameters.OBS_LATITUDE, unit=u.deg)

```

Comment:
pass the latitude as an argument, with a if to check its type. If its type is a string, convert it, if it is already an astropy.coordinates, then fine.

Reply:
done